src/vaadin-details.html

```diff
@@ -125,12 +125,17 @@
           };
         }
 
+        /**
+         * @return {Element}
+         * @protected
+         */
         get _collapsible() {
           return this.shadowRoot.querySelector('[part="content"]');
         }
 
         /**
          * Focusable element used by vaadin-control-state-mixin
+         * @return {Element}
```

Comment:
`ControlStateMixin` defines this as:
```ts
readonly focusElement: Element|null|undefined;
```

But it should be ok to be more explicit here, since we know that here this always returns an HTMLElement (which is assignable to Element).

```suggestion
         * @return {!HTMLElement}
```

I tested that this compiles without errors (if using other type that isn't assignable to `Element|null|undefined` then there's a compilation error).

Reply:
Done.